scripts/build-controller-image.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-set -E
+set -euxo pipefail
```

Comment:
Hi @tbutler4! Sorry for the delayed review on this!

The addition of the `-x` option will make the default output of the scripts extremely verbose, so I think that should be hidden behind a conditional on the `DEBUG` environs variable inside `scripts/lib/common.sh`:

```bash
if [[ "${DEBUG}" = "true" ]]; then
    set -x
fi
```

For the `-u` option, unfortunately there are a number of scripts that assume the presence of variables, but they do so in a "safe" manner by encapsulating Bash tests for the value of a potentially unset variable with `"${SOME_VAR}" = "someval"` like I've done in the code above. The `-u` option will just print a whole bunch of error messages that really aren't errors, so I'm afraid you'll need to remove that from the option set.

Reply:
Hi @jaypipes,

With all the exciting news from reInvent, I appreciate how you made time to offer support. So thank you!!

Your suggestions have helped me learn more on how to maneuver through bash scripts. I believe everything is correct now but please let me know if any additionally changes are necessary.